src/App/pages/ReposSearchPage/ReposSearchPage.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import Button from "@components/Button";
+import Input from "@components/Input";
+import RepoTile from "@components/RepoTile/RepoTile";
+import SearchIcon from "@components/SearchIcon";
+import ReactDOM from "react-dom";
+import { RepoItem } from "src/store/GitHubStore/types";
+
+import { ApiResponse } from "../../../shared/store/ApiStore/types";
+import GitHubStore from "../../../store/GitHubStore";
+
+function ReposSearchPage() {
+  const [inputValue, setInputValue] = React.useState("");
+  const [repoList, setRepoList] = React.useState<Array<any>>();
+  const [isLoading, setIsLoading] = React.useState(false);
+  React.useEffect(() => {
+    renderReply(repoList);
+  }, [repoList]);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) =>
+    setInputValue(e.target.value);
+
+  const renderReply = (reply: Array<any> | undefined) => {
```

Comment:
не понимаю зачем, можно же просто отренедерить и не надо использовать `ReactDOM.render`
просто ниже
```suggestion
  <div className="repo-list">
  {repoList.map((repo) => {
            return (
              <RepoTile
                key={repo.id}
                title={repo.name}
                organization={repo.owner.login}
                starCounter={repo.stargazers_count}
                updated={repo.updated_at}
              />
            );
          })}
  </div>
```